src/saving/SaveHelper.cs

```diff
@@ -128,6 +142,7 @@ public static void LoadSave(string name)
 
         GD.Print("Starting load of save: ", name);
         new InProgressLoad(name).Start();
+        lastSave = DateTime.Now;
```

Comment:
Actually now that I'm looking at this gain, these time resets should be in the InProgressLoad and InProgressSave classes on the final step, because it can take up to like 5 seconds to load a save now (and likely more in the future), so this time is way too early.